cluster/evaluator/launch_eval.py

```diff
@@ -70,35 +75,68 @@ def same_run_eval(black_num=0, white_num=0):
 
     b = fsdb.get_model(black_num)
     w = fsdb.get_model(white_num)
-    bucket = fsdb.eval_dir
 
     b_model_path = os.path.join(fsdb.models_dir(), b)
     w_model_path = os.path.join(fsdb.models_dir(), w)
 
     launch_eval_job(b_model_path + ".pb",
-               w_model_path + ".pb",
-               "{:d}-{:d}".format(black_num, white_num),
-               bucket)
+                    w_model_path + ".pb",
+                    "{:d}-{:d}".format(black_num, white_num),
+                    flags.FLAGS.bucket_name)
+
+
+def _append_pairs(new_pairs, dry_run):
+    desired_pairs = restore_pairs() or []
+    desired_pairs += new_pairs
+    print("Adding %d new pairs" % len(new_pairs))
+    if not dry_run:
+        save_pairs(desired_pairs)
+
```

Comment:
I'd have _append_pairs, add_*_pairs return desired_pairs 
so that you don't have to immediately reload the file to see new pairs on line 157

Reply:
ack.  These are also run as standalone commands, so 'returning' to bash won't help.